src/Altinn.Apps/KubernetesWrapper/deployment/values.yaml

```diff
@@ -0,0 +1,33 @@
+replicaCount: 1
+
+image:
+  # Set "repository" name of your image for manual Helm install and upgrade.
+  repository: tddregistry.azurecr.io/kuberneteswrapper
```

Comment:
Shouldn't this docker image be in a non-environment-dependent container registry? Or are we going to push this image to every registry we create for every environment?

Reply:
Very good point. We should have a shared container registry for all altinn apps, where applications that are shared among all app clusters should be shared. This has been noted as a follow up task in #1344.